CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # App Center SDK for React Native Change Log
 
-## Version 4.0.3 (Under development)
+## Version 4.1.0
```

Comment:
You must copy related changes from native SDKs

Reply:
Updated CHANGELOG for this, please help to check.

Reply:
Don't we need to add these changes about Android.Crashes too?
https://github.com/microsoft/appcenter-sdk-android/blob/develop/CHANGELOG.md#app-center-crashes
```
[Fix] Fix formatting of stack trace in the ErrorReport.
[Fix] Fix setting userId value in NDK crashes before sending.
```